ui/pages/confirmations/components/gas-timing/gas-timing.component.js

```diff
@@ -130,12 +132,13 @@ export default function GasTiming({
 
   const estimateToUse =
     estimateUsed || transactionData.userFeeLevel || 'medium';
-  let text = t(estimateToUse);
+  const estimateEmoji = PRIORITY_LEVEL_ICON_MAP[estimateToUse];
+  let text = `${estimateEmoji} ${t(estimateToUse)}`;
   let time = '';
   let attitude = 'positive';
 
   if (estimateToUse === 'low') {
-    text = t('gasTimingLow');
+    text = `${estimateEmoji} ${t('gasTimingLow')}`;
```

Comment:
We may need to confirm that it does not changes old confirmations.

Reply:
yes, this came up during design review. I'm checking with @bschorchit wether or not this can ship to old confirmations